App/App_iOS.swift

```diff
@@ -50,6 +45,24 @@ struct Kiwix: App {
                         NotificationCenter.openURL(url)
                     }
                 }
+                .task {
+                    if FeatureFlags.hasLibrary {
+                        fileMonitor.start()
+                        LibraryOperations.reopen {
+                            navigation.navigateToMostRecentTab()
+                        }
+                        LibraryOperations.scanDirectory(URL.documentDirectory)
+                        LibraryOperations.applyFileBackupSetting()
+                        LibraryOperations.applyLibraryAutoRefreshSetting()
+                        DownloadService.shared.restartHeartbeatIfNeeded()
+                    } else if let url = Brand.mainZimFileURL {
+                        LibraryOperations.open(url: url) {
+                            navigation.navigateToMostRecentTab()
+                        }
+                    } else {
+                        assertionFailure("App should support library, or should have a main zip file")
```

Comment:
The test here are not explicit enough IMO. We clearly need a primitive isCustomApp()  - or similar - which returns `true` or `false` and act accordingly. This primitive would be then used in many places. I would keep the `FeatureFlags.has...()` only for options which can be `true` or `false` within a custom app, depending how the custom app is configured. Doing so would allow to avoid the last `else` as well.

Reply:
I see, I think it can be grouped together, although the last ``else`` statement will still be present, in a way. 
That is more for the case if we configure the custom app in a wrong way, eg. if the zim file cannot be read (eg. wrong filename or permission issues).

Reply:
I managed to simplify this config for the Kiwix app, now it only contains the bare minimum, in the info.plist file and in the xccoonfig. Additionally I managed to cut down on this if else, as this validation is now managed in one place.